Module-5/helpers/POSTHandlers.js

```diff
@@ -0,0 +1,13 @@
+import { createUser } from './helpers.js';
+
+export const handleCreateUser = (req, res) => {
```

Comment:
this file is a `controller`

ideally, I would categorize your requests based on domains, rather than HTTP methods
it is waaay better to look for the controller based on the domain, rather than opening controllers on HTTP methods